_docs/style/text-conventions/capitalization.md

```diff
@@ -116,11 +116,11 @@ The following elements use sentence-style capitalization:
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
 
-## All-caps capitalization
+## Uppercase capitalization
 
-In rare cases, all-caps capitalization might be used to visually distinguish a secondary text element, such as, **GENERAL** as a menu heading.
+In rare cases, uppercase capitalization might be used to visually distinguish a secondary text element, such as, **GENERAL** as a menu heading.
```

Comment:
on further review, this sentence doesn't make sense. what does the general thing mean? where does that show up? 

We should also (maybe) give a bit of guidance about how to use uppercase and point to the difficulty reading.  Maybe change to something like:

"In rare cases, uppercase capitalization might be used to visually distinguish a text element from the text around it. Uppercase text style is difficult to read, so use it sparingly, and only for very short (1-3 word) uses." 

the tone might not be right, but the point is don't use it for sentences and avoid entirely if you can.